src/core/lib/security/authorization/sdk_server_authz_filter.cc

```diff
@@ -107,14 +66,14 @@ bool SdkServerAuthzFilter::CallData::IsAuthorized(SdkServerAuthzFilter* chand) {
         std::string(args.GetPeerAddressString()).c_str(), args.GetPeerPort());
   }
   grpc_authorization_policy_provider::AuthorizationEngines engines =
-      chand->provider_->engines();
+      provider_->engines();
   if (engines.deny_engine != nullptr) {
     AuthorizationEngine::Decision decision =
         engines.deny_engine->Evaluate(args);
     if (decision.type == AuthorizationEngine::Decision::Type::kDeny) {
       if (GRPC_TRACE_FLAG_ENABLED(grpc_sdk_authz_trace)) {
-        gpr_log(GPR_INFO, "chand=%p calld=%p: request denied by policy %s.",
```

Comment:
Hmm.  In the promise world, is there no longer any identifier to represent the individual call?  Losing this may make logs harder to understand, because if there are multiple calls in flight, it won't be clear which log messages are associated with which calls.

It's not so much a problem in this particular filter, since all of the per-call log messages are going to be logged in the same thread.  But there are other filters that do async work where this will be a problem.

Reply:
There's absolutely an identifier: the current activity pointer would probably make a good one (though we're not emulating that just yet) - and that's probably better than the call data pointer of some arbitrary filter because it's stable across the stack.

Reply:
That sounds like a nice improvement!  However, if we remove the calld pointer from the logs as we convert each filter, won't we then have to go back and change all of the log messages in every filter after we've finished converting to promises?  I'm wondering if there's a way we can structure this such that we keep the log messages up to date as we go.